### PR TITLE
Shivlaks/move cdk explorer

### PR DESCRIPTION
**Pull request by shivlaks**
## Description

This change is to move the AWS CDK explorer from it's own container in the activity bar into the main Explorer container. This also adds a configuration setting to allow users the capability to disable the explorer.

Finally, updating the help nodes to point to documentation and for providing feedback (This currently links to the issue template).

## Motivation and Context

It's idiomatic for CDK developers to be able to view a design-time view of their code without having to switch contexts. This explorer will be enabled by default. Developers will have a mechanism to disable it through a configuration setting.

## Related Issue(s)

Following-up on the discussion initiated in #809 

## Testing

* Tested it locally by launching the explorer
* Opened the 

## Screenshots (if appropriate)
Screen Shot 2019-11-20 at 11 04 55 AM
Screen Shot 2019-11-20 at 11 04 35 AM
Screen Shot 2019-11-20 at 11 04 21 AM

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the changelog using the script `npm run newChange`

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -144,6 +151,18 @@ export async function activate(context: vscode.ExtensionContext) {
                 await showQuickStartWebview(context)
             }
         })
+        registerCommand({
```
Comment (awschristou) [inline src/extension.ts]: These would go into the register commands portion of the cdk's activation file to keep a feature's activation code together

Comment (shivlaks) [reply src/extension.ts]: Makes sense. I get the reorg of activation a little better now :)
updating!

File: package.json [hunk 2]
```diff
@@ -80,6 +80,11 @@
                     "default": 30000,
                     "markdownDescription": "%AWS.configuration.description.samcli.debug.attach.timeout%"
                 },
+                "aws.cdk.explorer.enabled": {
```
Comment (awschristou) [inline package.json]: Food for thought: In CDK activation, the [API](https://code.visualstudio.com/api/references/vscode-api#workspace) `onDidChangeConfiguration` could be used against this configuration value changing in order to generate metrics.

Comment (shivlaks) [reply package.json]: I like that suggestion... will start looking into using it

**Pull request merged**